Bottom: dd36ca0f7b404195370e44072ea27a17ed692540
Top:    02ca33fa304ec095957b9f05dee0f7d6380bbebd
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-11-14 14:54:14 +0800

Refresh of vhost_net-support-both-xdp-and

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index b38aaff..9b1a072 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -837,7 +837,7 @@ static void handle_rx(struct vhost_net *net)
 		/* On error, stop handling until the next kick. */
 		if (unlikely(headcount < 0))
 			goto out;
-		if (nvq->xdp_ring)
+		if (nvq->rx_array)
 			msg.msg_control = vhost_net_buf_consume(&nvq->rxq);
 		/* On overrun, truncate and discard */
 		if (unlikely(headcount > UIO_MAXIOV)) {
